src/components/common/Sidebar/Sidebar.js

```diff
@@ -171,6 +174,9 @@ const Sidebar = ({ children, userProfile }) => {
               <Menu.Item key="/reviews" onClick={handleMenuClick}>
                 View Reviews
               </Menu.Item>
+              <Menu.Item key="/addMentorReview" onClick={handleMenuClick}>
+                Add Mentor Review
+              </Menu.Item>
```

Comment:
I'm confused. Why is /addMentorReview here twice?

Reply:
One is under 'Is user a mentee' and one is under 'is user an admin' so that It renders if you are either a mentee or an admin. I have it set to admin as well just so I could see it. I think it would be okay if the admin can see it. It could also just be deleted with no problem.